buildres/windows/JabRef.wxs

```diff
@@ -83,5 +133,6 @@
     <WixVariable Id="WixUILicenseRtf" Value="$(var.JpLicenseRtf)"/>
     <?endif?>
 
-  <DirectoryRef Id="TARGETDIR"><Component Id="RegistryJabFoxEntries" Guid="b6bc55ad-905c-4258-89b1-8b37abbe559c" Win64="yes"><RegistryKey Root="HKMU" Key="SOFTWARE\Mozilla\NativeMessagingHosts\org.jabref.jabref" Action="createAndRemoveOnUninstall" ForceCreateOnInstall="yes"><RegistryValue Type="string" Value="[APPLICATIONFOLDER]jabref.json"/></RegistryKey></Component></DirectoryRef><Feature Id="JabFox" Level="1"><ComponentRef Id="RegistryJabFoxEntries" /></Feature><Property Id="ARPPRODUCTICON" Value="DesktopIcon.exe" /><WixVariable Id="WixUIBannerBmp" Value="JabRefTopBanner.bmp" /><DirectoryRef Id="TARGETDIR"><Component Id="RegistryJabFoxEntries" Guid="b6bc55ad-905c-4258-89b1-8b37abbe559c" Win64="yes"><RegistryKey Root="HKMU" Key="SOFTWARE\Mozilla\NativeMessagingHosts\org.jabref.jabref" Action="createAndRemoveOnUninstall" ForceCreateOnInstall="yes"><RegistryValue Type="string" Value="[APPLICATIONFOLDER]jabref.json"/></RegistryKey></Component></DirectoryRef><Feature Id="JabFox" Level="1"><ComponentRef Id="RegistryJabFoxEntries" /></Feature><Property Id="ARPPRODUCTICON" Value="DesktopIcon.exe" /><WixVariable Id="WixUIBannerBmp" Value="JabRefTopBanner.bmp" /><DirectoryRef Id="TARGETDIR"><Component Id="RegistryJabFoxEntries" Guid="b6bc55ad-905c-4258-89b1-8b37abbe559c" Win64="yes"><RegistryKey Root="HKMU" Key="SOFTWARE\Mozilla\NativeMessagingHosts\org.jabref.jabref" Action="createAndRemoveOnUninstall" ForceCreateOnInstall="yes"><RegistryValue Type="string" Value="[APPLICATIONFOLDER]jabref.json"/></RegistryKey></Component></DirectoryRef><Feature Id="JabFox" Level="1"><ComponentRef Id="RegistryJabFoxEntries" /></Feature><Property Id="ARPPRODUCTICON" Value="DesktopIcon.exe" /><WixVariable Id="WixUIBannerBmp" Value="JabRefTopBanner.bmp" /></Product>
```

Comment:
We need to add this JabFox stuff as well I think

Reply:
This should be done by the post-inst script. Let's see if it works